### PR TITLE
revised code and documentation.

### PR DESCRIPTION
**Pull request by jakejh**
Most of the changes are minor: clarifying documentation and simplifying code. Please look through them carefully, though.

### CODE REVIEW

**Comment by jakejh** [comment]
Will close #72 and #69.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Version: 0.0.0.9000
-Authors@R: as.person(c(
-  "Jake Hughey <jakejhughey@gmail.com> [aut, cre]",
-  "Layla Aref <larefnezhad@gmail.com> [aut]"))
-Description: More about what it does.
+Version: 0.0.1
+Authors@R: c(
+  person('Jake', 'Hughey', , 'jakejhughey@gmail.com', c('aut', 'cre')),
+  person('Layla', 'Aref', , 'larefnezhad@gmail.com', 'aut'))
+Description: Use phenotype risk scores based on linked clinical and genetic data to study Mendelian disease and rare genetic variants.
```
Comment (JSchoenbachler) [inline DESCRIPTION:8]: @jakejh do we want to add the DOI for the first phers paper?

File: R/genetic_association.R [hunk 2]
```diff
@@ -49,92 +61,48 @@ getGeneticAssociations = function(
...
-
-    # exclude variants with only one genotype in the population
-    genoCount = melt(
-      genotypesSub[, lapply(.SD, uniqueN, na.rm = TRUE), .SDcols = snpSub],
-      measure.vars = snpSub, variable.name = 'snp', value.name = 'count',
-      variable.factor = FALSE)
-    snpSub = unique(genoCount[count != 1]$snp)
+      person_id = rownames(genotypes), genotypes[, snpSub, drop = FALSE])
 
+    statsSnps = foreach(snp = snpSub, .combine = rbind) %do% {
```
Comment (JSchoenbachler) [inline R/genetic_association.R:87]: I'm not sure about this, but couldn't you just remove `statsSnps = ` and the output of the inner foreach would directly pipe to the output of the outer foreach?

**Comment by JSchoenbachler** [comment]
@jakejh just added a few comments while looking over, looks good to me!

File: R/genetic_association.R [hunk 3]
```diff
@@ -1,46 +1,58 @@
...
 #' @eval example3()
 #'
+#' @seealso [stats::glm()], [stats::confint()], [getScores()], [phers()]
+#'
 #' @export
 getGeneticAssociations = function(
   scores, genotypes, demos, diseaseVariantMap, glmFormula,
-  modelType = c('additive', 'dominant', 'recessive', 'genotypic'),
-                level = 0.95, dopar = FALSE) {
+  modelType = c('genotypic', 'additive', 'dominant', 'recessive'),
```
[the remark this answers is not in the document]
Comment (jakejh) [reply R/genetic_association.R:53]: Based on conversations with Lisa, we want genotypic to be the default.

File: R/genetic_association.R (hunk 2 above)
Comment (jakejh) [reply R/genetic_association.R:87]: Yes, but it makes development easier to explicitly assign the result to a variable.

File: DESCRIPTION (hunk 1 above)
Comment (jakejh) [reply DESCRIPTION]: Good call.

File: R/genetic_association.R (hunk 3 above)
[the remark this answers is not in the document]
Comment (JSchoenbachler) [reply R/genetic_association.R:53]: Gotcha, so default first then alphabetic asc ordering.

**Pull request merged**